src/wpanctl/tool-cmd-commissioner.c

```diff
@@ -305,7 +305,7 @@ int tool_cmd_commissioner(int argc, char* argv[])
 					goto bail;
 				}
 				if (strnlen(ext_addr, EXT_ADDRESS_LENGTH_CHAR+1) != EXT_ADDRESS_LENGTH_CHAR) {
-					fprintf(stderr, "%s: error: Wrong address length.%d \n", argv[0], strnlen(ext_addr, EXT_ADDRESS_LENGTH_CHAR+1));
+					fprintf(stderr, "%s: error: Wrong address length.%zu \n", argv[0], strnlen(ext_addr, EXT_ADDRESS_LENGTH_CHAR+1));
```

Comment:
The problem is that `strnlen` returns a `size_t`, which may be a `unsigned long long` or an `unsigned long` or even an `unsigned int`, depending on the platform. This means that there isn't a good universal way to handle the situation without casting.